### DISCUSSION TITLE
PCBWay order

### DISCUSSION THREAD

**Discussion by RadAd** [picocomputer/community#18] [Q&A]
Hi

FIrstly, thanks rumbledethumps! This is a great project. I have followed a few other projects like this, but I like your approach of using the Raspberry Pi Pico for the VGA and io.

I am investigating getting an assembled pcb from pcbway. I live in Melbourne, Australia and I was wondering if anyone would like to share the costs. If I order one for myself it comes out to $135 AUD. But if I order 5 it comes out to about $55 AUD for each board.

The extra parts of the two Raspberry Pi Picos and the other chips are not included in that price. I could look into whether there are any savings there also.

So are there any locals also considering getting one and maybe we can share the costs?

Adam.

**Comment by snakebyte69** [comment]
I'm based in Australia and in the process of manufacturing boards, you can pre-order one here at a much cheaper price than $135
https://agonlight.au

**Reply by linuxplayground** [reply]
@snakebyte69 Ah! I just saw this. I'll update the message I wrote in general.  Any chance you could have a look at the messages I've sent to you via Tindie?

[Question in reference to Order #421633 placed 09/29/2023]

I ordered with shipping to new Zealand and auspost status is stuck in Pending for over 10days now.

**Reply by linuxplayground** [reply]
I've heard from Agon Australia now. Thanks.

**Reply by linuxplayground** [reply]
@snakebyte69 thanks for getting back to me last week. Although you mentioned that you would be shipping the kit, the tracking is still showing in pending status. Please can you provide an update?
Tindie order number: 421633

Thanks
Dave Latham
